### ISSUE TITLE
Upgrade Rswag to 2.13

### ISSUE THREAD

**Issue by stevenjcumming** [department-of-veterans-affairs/va.gov-team#72416]
## Description
[Rswag] extends rspec-rails "request specs" with a Swagger-based DSL for describing and testing API operations. 2.11 introduced a deprecation warning that `rswag`was switching to [OpenApi Specification 3](https://swagger.io/docs/specification/api-host-and-base-path/)(OAS 3) to stay current with Swagger. The deprecation warning occurs because `basePath` is removed from OAS 3. 

---

## Tasks
- [x] Upgrade `rswag-specs` and `rswag-ui ` to 2.13 
- [x] Remove `basePath` from RswagConfig files
- [x] Update swagger_* methods in `swagger_helper.rb` (spec/swagger_helper.rb) to use OAS 3 compatible methods
- [x] Update rswag documentation

## Acceptance Criteria
- [x] `rswag-specs` and `rswag-ui ` is  v2.13 
- [x] AppealsApi and ClaimsApi are accessible/working locally via swagger-ui
- [x] Resolve rswag deprecation warnings in CI checks
- [x] swagger docs are OAS 3 compliant 

## Notes
- Upgrading to 2.13 unblocks `json-schema` from being upgraded. 
- Based on comments and the method `rswag_to_oas!`, `basePath` was used to generated swagger files but removed in the `rswag.rake` task for AppealsApi, but not ClaimsApi. 
- This doesn't apply to requests that use [swagger-blocks](https://github.com/fotinakis/swagger-blocks). Which will need a separate issue/PR.

**Comment by ericboehs** [comment]
This was merged on Tuesday (last sprint).
